packages/web-react/src/components/FileUploader/FileUploaderAttachment.tsx

```diff
@@ -5,14 +5,29 @@ import { useStyleProps } from '../../hooks';
 import { useFileUploaderStyleProps } from './useFileUploaderStyleProps';
 import { useFileUploaderAttachment } from './useFileUploaderAttachment';
 import AttachmentDismissButton from './AttachmentDismissButton';
+import AttachmentImagePreview from './AttachmentImagePreview';
 import { Icon } from '../Icon';
 
 const FileUploaderAttachment = (props: SpiritFileUploaderAttachmentProps) => {
-  const { label, name, iconName = 'file', id, onDismiss, buttonLabel = 'Remove', file, onError, ...restProps } = props;
+  const {
+    label,
+    name,
+    iconName = 'file',
```

Comment:
Nice-to-have: Could you, please, do a little refactor and move this and a `Remove` few lines down to the `constants` file here using something like `DEFAULT_ICON_NAME` and the same for the button label? Thanks a lot.